### PR TITLE
Modify accommodations merging

### PR DESCRIPTION
**Pull request by oleg1std**
Modify accommodations merging

### CODE REVIEW

File: Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 {
-    public readonly struct CombineAccommodationsRequest
+    public readonly struct MergeAccommodationsRequest
     {
         [Required]
         public string BaseHtAccommodationId { get; init; } 
         
         [Required]
-        public string CombinedHtAccommodationId { get; init; }
+        public string MergedHtAccommodationId { get; init; }
```
Comment (hasmikharutyunyan) [inline Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs:11]: HtAccommodationIdToMerge

File: Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 namespace HappyTravel.Edo.Api.AdministratorServices.Models.Mapper
 {
-    public readonly struct CombineAccommodationsRequest
+    public readonly struct MergeAccommodationsRequest
```
Comment (hasmikharutyunyan) [inline Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs:5]: `AccommodationsMergeRequest`  ?

File: HappyTravel.Edo.Common/Enums/Administrators/AdministratorPermissions.cs [hunk 3]
```diff
@@ -15,14 +15,13 @@ public enum AdministratorPermissions
         CounterpartyToAgencyTransfer = 128,
         CounterpartyManagement = 256,
         PaymentLinkGeneration = 512,
-        AccommodationDuplicatesReportApproval = 1024,
+        AccommodationDuplicatesMerge = 1024,
```
Comment (hasmikharutyunyan) [inline HappyTravel.Edo.Common/Enums/Administrators/AdministratorPermissions.cs:18]: Maybe just `AccommodationsMerge` ? I think word `Duplicate` we will use in another places

File: Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs (hunk 1 above)
Comment (oleg1std) [reply Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs]: fixed

File: HappyTravel.Edo.Common/Enums/Administrators/AdministratorPermissions.cs (hunk 3 above)
Comment (oleg1std) [reply HappyTravel.Edo.Common/Enums/Administrators/AdministratorPermissions.cs]: fixed

File: Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs (hunk 2 above)
Comment (oleg1std) [reply Api/AdministratorServices/Models/Mapper/MergeAccommodationsRequest.cs]: fixed

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**